### ISSUE TITLE
FO - Images are not well displayed when create a product with V2 page 

### ISSUE THREAD

**Issue by HanaRebaiQA** [PrestaShop/PrestaShop#28338]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Images are not well displayed when create product with page V2

**Product details**

![image](https://user-images.githubusercontent.com/92912932/165300002-8fee1540-e4de-4f45-b8eb-e47cfa21bf48.png)

**Order confirmation**

![image](https://user-images.githubusercontent.com/92912932/165300009-d4ca9e95-e8dc-4d8f-9967-b7b9cd3e3ef0.png)

https://user-images.githubusercontent.com/92912932/165300449-d5ed3efa-b4f9-42db-99da-d58837f2a6a6.mp4

### Expected behavior

All images are well diplayed

### Steps to reproduce

1. Create a product with page V2 (eg : Standard)
2. Add an image for this product
3. GO to FO > Search for the product and click on it
4. See error > The image is not well displayed

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

7.4

**Comment by zuk3975** [comment]
As described in duplicate issue https://github.com/PrestaShop/PrestaShop/issues/29065 I also reproduced the issue only in develop, but 8.0.x seems to work.

So original cause of this seems to be empty link rewrite. It seems to be fixed in 8.0.x by https://github.com/PrestaShop/PrestaShop/pull/28553/files so it will automatically get fixed when 8.0.x is merged to develop.

I found another issue though (in 8.0.x too). When product has no name and I try to enable it in the list, it doesn't show any error. (I see error in console, but we should also show the error in front end smth like - `You cannot enable product without name & friendly url`)
